packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -11,6 +11,7 @@
   --terra-consumer-profile-link-width: var(--terra-consumer-nav-width);
   --terra-consumer-profile-link-background: var(--terra-consumer-body-background);
   --terra-consumer-profile-link-color: var(--terra-consumer-nav-item-color);
+  --terra-consumer-logo-text-color: var(--terra-consumer-nav-item-color);
```

Comment:
We don't need a logo text color prop... it should just be the text color of the nav

Reply:
We can merge this for now and defer this when we do custom prop cleanup...